src/backend/src/services/change-requests.services.ts

```diff
@@ -104,8 +110,10 @@ export default class ChangeRequestsService {
     if (foundCR.wbsElement.dateDeleted) throw new DeletedException('WBS Element', wbsPipe(foundCR.wbsElement));
 
     // verify that the user is not reviewing their own change request
-    if (reviewer.userId === foundCR.submitterId) throw new AccessDeniedException();
-
+    if (reviewer.userId === foundCR.submitterId)
+      throw new AccessDeniedException(
+        `User ${reviewer.userId} is not allowed to review their own change request submitted by User ${foundCR.submitterId}`
+      );
     // If approving a scope CR
     if (foundCR.scopeChangeRequest && accepted) {
       // ensure a proposed solution is being approved and exists
```

Comment:
change this, if no wbs proposed changes and no psid, then error
if no proposedsolutions, but proposedchanges: do your new stuff
if no proposedchanges, but proposedsolutions: do old stuff